test/modules/MetricsValidator/src/Program.cs

```diff
@@ -38,6 +38,8 @@ static async Task<int> MainAsync()
                     .Build();
 
                 var transportType = configuration.GetValue("ClientTransportType", Microsoft.Azure.Devices.Client.TransportType.Mqtt);
+
+                Logger.LogInformation("Make Client");
```

Comment:
More specific, e.g. Metrics Validator create module client instance. etc.

Reply:
This will only ever show up in MetricsValidator logs, do I really need to be that specific? It's like 2 lines down from "Starting Metrics Validator" in the logs

Reply:
at least put "Use module client" sth, "Client" is too general.